src/layout/GridList.jsx

```diff
@@ -40,19 +77,17 @@ export class GridListComponent extends React.Component {
 			{
 				'flex--wrap': autoHeightWithWrap,
 				[`${GRID_AUTOHEIGHT_CLASS}--has${columns.default}`]: !!columns.default,
-				[`atMedium_${GRID_AUTOHEIGHT_CLASS}--has${
-					columns.medium
-				}`]: !!columns.medium,
-				[`atLarge_${GRID_AUTOHEIGHT_CLASS}--has${
-					columns.large
-				}`]: !!columns.large,
+				[`atMedium_${GRID_AUTOHEIGHT_CLASS}--has${columns.medium ||
+					0}`]: !!columns.medium,
+				[`atLarge_${GRID_AUTOHEIGHT_CLASS}--has${columns.large ||
+					0}`]: !!columns.large,
 			},
 			className
 		);
 
 		const listItemClassNames = cx('gridList-item', {
 			['flex-item']: autoHeight,
-			[itemClassNames]: itemClassNames,
+			[itemClassNames || '']: Boolean(itemClassNames),
```

Comment:
`|| ''` shouldn't be necessary here because when `itemClassNames` is an empty string or undefined this will return an empty string

Reply:
this was also to appease flow, i'll fix this